aptos-move/framework/aptos-framework/sources/jwks.move

```diff
@@ -201,6 +202,29 @@ module aptos_framework::jwks {
         assert!(num_bytes < MAX_FEDERATED_JWKS_SIZE_BYTES, error::invalid_argument(EFEDERATED_JWKS_TOO_LARGE));
     }
 
+    // This can be called to install or update a set of JWKs for a federated OIDC provider.  This function should
+    // be invoked to intially install a set of JWKs or to update a set of JWKs when a keypair is rotated.
+    public entry fun update_federated_jwk_set(jwk_owner: &signer, iss: vector<u8>, kid_vec: vector<String>, alg_vec: vector<String>, e_vec: vector<String>, n_vec: vector<String>) acquires FederatedJWKs {
```

Comment:
Which AIP does this apply to?

Reply:
[AIP-96](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-96.md).

@heliuchuan, add to comments?